### ISSUE TITLE
Cant open OpenShot without internet connection, Lubuntu 20.04.

### ISSUE THREAD

**Issue by izzostar** [OpenShot/openshot-qt#5088]
**Describe the bug**
Cant open OpenShot without internet connection, Lubuntu 20.04.

**Steps to reproduce the behavior:**
1. Go to vacation
2. Click on hotel wifi
3. Scroll down to insert password for hotel wifi in firefox.
4. Lose the wifi connection.
5. Try open OpenShot for editing videos.
6. Openshot window wont appear, but openshot is listed in task manager.

**Expected behavior**
Openshot opens.

**System Details**
- OpenShot Version: 3.0.0
- libopenshot Version:  0.3.0
- Platform: Linux-5.4.0-135-generic-x86_64-with-glibc2.29
- Processor: x86_64
- Machine: x86_64
- Python version: 3.8.10
- Qt5 version: 5.12.8
- PyQt5 version: 5.14.1
- Qt Detected Languages: ['en-US']
- LANG Environment Variable: en_US.UTF-8
- LOCALE Environment Variable: 
- Daily Build: Not able to verify issue still exists in daily build

**Log Files**
- [openshot-qt.log](https://s3.amazonaws.com/logs.openshot.org/q4ngfmca3rdnqpprulzl9bi2jmrbmkpd/openshot-qt.zip) (5425 KB)
- [libopenshot.log](https://s3.amazonaws.com/logs.openshot.org/q4ngfmca3rdnqpprulzl9bi2jmrbmkpd/libopenshot.zip) (9 KB)

**Exception / Stacktrace**
```No stacktrace found in log files```

**Screenshots** (Optional)
If applicable, add screenshots to help explain your problem. You can include screenshots by
copy/pasting them on GitHub or dragging-and-dropping into the GitHub page. All images are public,
so please don't post screenshots containing personal information.

**Comment by Colorjet3** [comment]
Just to be clear:
1.  OpenShot was working when connected to the internet.
2.  When you start OpenShot or you clicking on the OpenShot shortcut or are you clicking on an .OSP file?
3.  When you establish your wi-fi connection in the hotel, does OpenShot work?

**Comment by izzostar** [comment]
1. yes, at home it was working fine. may-be even without internet before connecting to the hotel.
2. Opening the program from Lubuntu start-menu. After not succeeded, also tried from the project file, still no go.
3. Yes, after having a wifi connection, it started working again, i can open the OpenShot program again. (before trying again i killed it from task-manager for just in case, also i cant remember if it was the same hotel wifi now or another one from my phone hotspot)
My wife uses this OpenShot, so i cant try right now at home if it works here without Wi-Fi, she refuses to disconnect, because she wants no troubles before the project is rendered :D :(

**Comment by Colorjet3** [comment]
Hello @izzostar I've been testing both in Windows and Ubuntu, without an internet connection, and OpenShot always start without any issues.

I was thinking about your situation and I think this is a special case where this is what happens:
1.  You go to the hotel.
2.  Your laptop is actually connected to the Wi-fi.
3.  However, you still need to go through some process (whatever security the hotel has setup) to get passed their firewall.
4.  So, you decide not to do this for now and you start OpenShot.
5.  OpenShot sees that there a network connection but doesn't really know that there is no Internet access.
6.  So, whatever it is checking for causes issues.

To prove this right or wrong, what you can do the next time you are at this hotel or any other place where you can see the wi-fi but don't a have access to the Internet until you agree to the terms or provide some credentials, is to just disable your wi-fi on your laptop.  Then start OpenShot.  If it runs, than my theory is correct.  If it doesn't run then we are back to square 1.

**Comment by izzostar** [comment]
Yes, but why does it need internet anyway, its a video editor? And can i disable it from anywhere in the menu?

**Comment by Colorjet3** [comment]
It does not need internet to work.  I just can not figure out why your system is working that way.  This is very strange.

**Comment by BobSynfig** [comment]
Couldn't it be related with some network shortcuts registered in your Networks Places or even inside the project?
OpenShot may try to access some resources that can't be reached over the network and would be in waiting state during initialization?

May be related:
App freezes when no connection available https://github.com/OpenShot/openshot-qt/issues/5002
I didn't see any option to remove this check in the Preference.

Also there is one that could be disabled: Preferences/Debug/Send Anonymous Metrics and Errors

**Comment by cboltz** [comment]
Wild guess: could this be caused by #4735?
